doc/examples.rst

```diff
@@ -0,0 +1,19 @@
+:orphan:
+
+.. _examples:
+
+=======================
+User Guide and Examples
+=======================
+
```

Comment:
A little blurb describing what to expect here would be good. e.g.,

```RST
The following examples are `literate programming <https://en.wikipedia.org/wiki/Literate_programming>`__
documents which solve specific problems using Nipype.
In some cases, the same problem is solved with multiple underlying tools.
```

We could elaborate more, but this should be enough to orient.

Reply:
Added